swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -28,11 +31,16 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     // Initialize Telemetry as early as possible
     Telemetry.start()
 
+    guard let userDefaults = UserDefaults(suiteName: BundleHelper.appGroupId)
```

Comment:
It appears that a wrapper around `UserDefaults` could be a good idea to provide type-safe access to our config values.